packages/cli/src/playground/src/domains/workflows/workflow-information.tsx

```diff
@@ -91,12 +91,7 @@ export function WorkflowInformation({ workflowId, isLegacy }: { workflowId: stri
                   {isLegacy ? (
                     <LegacyWorkflowTrigger workflowId={workflowId} setRunId={setRunId} />
                   ) : (
-                    <WorkflowTrigger
-                      workflowId={workflowId}
-                      setRunId={setRunId}
-                      workflow={workflow}
-                      isLoading={isWorkflowLoading}
-                    />
+                    <WorkflowTrigger workflowId={workflowId} setRunId={setRunId} />
```

Comment:
ℹ️ This is for showcasing that it works. It rolled-back to a situation where playground UI actually fetches the data without knowing from where it came